exercises/resistor-color-duo/description.md

```diff
@@ -2,19 +2,9 @@ If you want to build something using a Raspberry Pi, you'll probably use _resist
 
 * Each resistor has a resistance value.
 * Resistors are small - so small in fact that if you printed the resistance value on them, it would be hard to read.
-To get around this problem, manufacturers print color-coded bands onto the resistors to denote their resistance values. Each band acts as a digit of a number. For example, if they printed a brown band (value 1) followed by a green band (value 5), it would translate to the number 15.
+To get around this problem, manufacturers print color-coded bands onto the resistors to denote their resistance values. Each band has a position and a numeric value. For example, if they printed a brown band (value 1) followed by a green band (value 5), it would translate to the number 15.
 
 In this exercise, you are going to create a helpful program so that you don't have to remember the values of the bands. The program will take two colors as input, and output the correct number.
 
-The band colors are encoded as follows:
-
-- Black: 0
-- Brown: 1
-- Red: 2
-- Orange: 3
-- Yellow: 4
-- Green: 5
-- Blue: 6
-- Violet: 7
-- Grey: 8
-- White: 9
+The colors are mapped to the numbers from 0 to 9 in the sequence:
```

Comment:
```suggestion
The colors are mapped to the significant digits, from 0 to 9, in the sequence:
```

I prefer using significant digits, because "number" it too abstract, and not really correct, e.g. brown (1) is `1 * 10 ^ 1` or `1 * 10 ^ 0`.

Reply:
The 'significants' was removed from a previous version, per request of the product team. 
@ihid, @kytrinyx, if you're okay with this, I'm okay with it too.

Reply:
Ah. Okay.

I think significant digits might be a pretty technical term, and _may_ be why it was removed :sweat_smile:, but perhaps just "digits" would work. The significant part follows from the rest of the README?